### PR TITLE
Adds valid brazilian CPF/CNPJ generators

### PR DESCRIPTION
**Pull request by Drowze**
Adds a simple algorithm for both brazilian CPFs and CNPJs.

### CODE REVIEW

File: lib/ffaker/identification_br.rb [hunk 1]
```diff
@@ -6,13 +6,31 @@ module IdentificationBR
...
+
+    def cnpj
+      cnpj = FFaker.numerify('############')
+      validators = cnpj_verification_digits(cnpj)
+      cnpj + validators.join
+    end
+
+    def pretty_cnpj
+      cnpj_numbers = cnpj.to_s.rjust(14, '0')
+      sprintf('%s.%s.%s/%s-%s', cnpj_numbers[0..1], cnpj_numbers[2..4], cnpj_numbers[5..7],
```
Comment (Drowze) [inline lib/ffaker/identification_br.rb]: Can I have some documentation on this?

File: lib/ffaker/identification_br.rb [hunk 2]
```diff
@@ -6,13 +6,31 @@ module IdentificationBR
...
+      cpf = FFaker.numerify('#########')
+      validators = cpf_verification_digits(cpf)
+      cpf + validators.join
     end
 
     def pretty_cpf
-      FFaker.numerify('###.###.###-##')
+      cpf_numbers = cpf
+      sprintf('%s.%s.%s-%s', cpf_numbers[0..2], cpf_numbers[3..5],
+                             cpf_numbers[6..8], cpf_numbers[9..10])
```
Comment (Drowze) [inline lib/ffaker/identification_br.rb]: Would 
```ruby
      sprintf('%s.%s.%s-%s', cpf_numbers[0..2], cpf_numbers[3..5],
       cpf_numbers[6..8], cpf_numbers[9..10])
```
be the right way?

File: lib/ffaker/identification_br.rb [hunk 3]
```diff
@@ -6,13 +6,38 @@ module IdentificationBR
...
+
+    def cnpj
+      cnpj = FFaker.numerify('############')
+      validators = cnpj_verification_digits(cnpj)
+      cnpj + validators.join
+    end
+
+    def pretty_cnpj
+      cnpj_numbers = cnpj.to_s.rjust(14, '0')
+      sprintf('%s.%s.%s/%s-%s',
```
Comment (Drowze) [inline lib/ffaker/identification_br.rb]: May I have an example on solving this?

Comment (marocchino) [inline lib/ffaker/identification_br.rb]: http://batsov.com/articles/2013/06/27/the-elements-of-style-in-ruby-number-2-favor-sprintf-format-over-string-number-percent/

Comment (Drowze) [inline lib/ffaker/identification_br.rb]: Fair enough! Corrected as requested.

**Comment by marocchino** [comment]
👍

**Pull request merged**